workflows/prognostic_c48_run/runtime/steppers/prescriber.py

```diff
@@ -136,7 +136,7 @@ def __call__(self, time, state):
                 # "ocean_surface_temperature". If "surface_temperature" is
```

Comment:
Nit: Rephrase comment to only focus on special case of setting ocean surface temperature. I see what you're saying with the latter part but it might be more distracting than helpful given that prescribing surface temperature isn't a use case we've encountered yet and is not on the prephysics 'whitelist'.